.circleci/config.yml

```diff
@@ -159,6 +159,16 @@ defaults: &defaults
     - image: cimg/node:14.16-browsers
   working_directory: ~/mapbox-gl-js
 
+use_ramdisk: &use_ramdisk
+  working_directory: /mnt/ramdisk/mapbox-gl-js
```

Comment:
Did you notice more consistent test running times with the ramdisk? Is it also possible to implement this on Mac and Windows?

Reply:
Here's an article about using a RAM Disk in Windows, a little more involved but worth a try https://support.circleci.com/hc/en-us/articles/4411520952091-Create-a-windows-RAM-disk

Reply:
It used to be a bit faster when I tried it before adding Windows and Mac, but now it's not so obvious, so we can also consider dropping this and not adding extra complexity.

Reply:
Thoughts on isolating it in a separate PR to better compare changes and not block the other improvements here?

Reply:
Sure, let's first land a flakiness fix and experiment with the RAM disk after.